pylossless/dash/topo_viz.py

```diff
@@ -33,6 +33,14 @@
 yaxis.update({"scaleanchor": "x", "scaleratio": 1})
 
 
+def pick_montage(montage, ch_names):
+    """Pick a subset of channels from a montage."""
+    digs = montage.remove_fiducials().dig
+    assert len(digs) == len(montage.ch_names)
```

Comment:
```suggestion
```

Reply:
I assume all of these asserts are just leftover from your debugging/development. We should delete them before merge. If we really want to check for these conditions we should probably use the `if not ... raise` pattern (the python interpreter can be told to ignore assert statements)